frontend/src/shared/components/register-page/register-page.component.html

```diff
@@ -0,0 +1,88 @@
+                                placeholder="Write password"
+                                [value]="password"
+                                [(ngModel)]="password"
+                                (ngModelChange)="resetPasswordControl()"
+                                formControlName="passwordControl"
+                              />
+                              <mat-icon class="hide-icon" (click)="hidePass = !hidePass">{{hidePass ? 'visibility' : 'visibility_off'}}</mat-icon>
+                        </div>                        
+                        <span class="error" *ngIf="passwordControl.errors?.['required'] && (passwordControl.dirty || passwordControl.touched)">Password is required.</span>
+                        <span class="error" *ngIf="passwordControl.errors?.['minlength']">Must be at least 8 characters</span>
+                    </div>
+                    <div class="input-container">
+                        <div class="label-container full-width">
+                            <label>Repeat password</label>
+                        </div>                        
+                        <div class="password-repeat-container">
+                            <input class="input full-width" [ngClass]="{
+                                'error': (passwordRepeatControl.invalid && (passwordRepeatControl.dirty || passwordRepeatControl.touched))
+                                }"
+                                [(ngModel)]="passwordRepeat"
+                                [type]="hidePassRepeat ? 'password' : 'text'" 
+                                placeholder="Repeat password"
+                                formControlName="passwordRepeatControl"
+                              />
+                              <mat-icon class="hide-icon" (click)="hidePassRepeat = !hidePassRepeat">{{hidePassRepeat ? 'visibilityRepeat' : 'visibility_off'}}</mat-icon>
+                        </div>                        
+                        <span class="error" *ngIf="passwordRepeatControl.errors?.['required'] && (passwordRepeatControl.dirty || passwordRepeatControl.touched)">Password repeat is required.</span>
+                        <span class="error" *ngIf="passwordRepeatControl.errors?.['pattern'] && (passwordRepeatControl.dirty || passwordRepeatControl.touched)">Password not match.</span>
+                    </div>
+                </form>
+            </div>
+            <div class="button-container">
+                <tasque-button text="Sign up" class="btn fill full-width" (btnClick)="submitForm()"></tasque-button>
+            </div>
+            <div class="button-container">
+                <tasque-button text="Sign up with Google" class="btn stroke full-width violet-stroke" [icon]="faGoogle"></tasque-button>
+            </div>
+            <div class="button-container">
+                <tasque-button text="Sign up with Github" class="btn full-width violet-stroke" [icon]="faGithub"></tasque-button>
```

Comment:
I don't think we need these buttons since we have them on login page and if user never registered we can register them immediately

Reply:
also you missed `stroke` class on gh button

Reply:
> I don't think we need these buttons since we have them on login page and if user never registered we can register them immediately

Let it be now, if these buttons will be unnecessary we can remove em in one minute
Stroke +